vector/src/main/java/im/vector/app/features/spaces/create/CreateSpaceViewModel.kt

```diff
@@ -204,12 +235,31 @@ class CreateSpaceViewModel @AssistedInject constructor(
                 }
                 _viewEvents.post(CreateSpaceEvents.NavigateToChoosePrivateType)
             } else {
+                // it'a public space, let's check alias
+                val aliasLocalPart = if (state.aliasManuallyModified) state.aliasLocalPart else getAliasFromName(state.name)
+                _viewEvents.post(CreateSpaceEvents.ShowModalLoading(null))
                 setState {
-                    copy(
-                            step = CreateSpaceState.Step.AddRooms
+                    copy(aliasVerificationTask = Loading())
+                }
+                viewModelScope.launch {
+                    session.checkAliasAvailability(aliasLocalPart).fold(
```

Comment:
I think it can crash if there is no network (see my previous remark)

Reply:
fixed